.pre-commit-config.yaml

```diff
@@ -25,13 +25,13 @@ repos:
         args: [--fix, --exit-non-zero-on-fix]
 
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v1.1.1
+    rev: v1.0.1
```

Comment:
Why exactly 1.0.1?

Reply:
Because that is the last version that falls into our otherwise supported range `mypy>=0.990,<1.1.0`